tox.ini

```diff
@@ -1,13 +1,18 @@
 [tox]
 envlist = py27,py34,py35,py36
```

Comment:
Please remove py27, py34, py35, py36 and add py37, py38, py39

Reply:
done